aptos-move/framework/aptos-stdlib/sources/fixed_point64.move

```diff
@@ -56,6 +56,19 @@ module aptos_std::fixed_point64 {
         ensures result.value == x.value + y.value;
     }
 
+    /// Returns x / y. The result cannot be greater than MAX_U128.
+    public fun divide(x: FixedPoint64, y: FixedPoint64): FixedPoint64 {
+        let x_raw = get_raw_value(x);
+        let y_raw = get_raw_value(y);
+        // If it is divisable, return the result. If not, return the result + 1.
+        let result = (x_raw as u256) / (y_raw as u256);
+        if ((x_raw as u256) % (y_raw as u256) != 0) {
+            result = result + 1;
+        };
+        assert!(result <= MAX_U128, ERATIO_OUT_OF_RANGE);
+        create_from_raw_value((result as u128))
+    }
```

Comment:
I don't think this division is correct. If both numbers have 64 bit allotted for fractional value, after division, the fraction would disappear. I believe `x_raw << 64` would be required before performing the division to ensure `result` still has 64 bit fractional bits.

Reply:
I think we don't even need it right now as we did not use this anymore. Do we need to keep this function?